.github/workflows/build_release_assets.yml

```diff
@@ -18,7 +18,8 @@ on:
         default: false
 
 env:
-  PDM_VERSION: 2.22.4
+  PDM_VERSION: 2.26.0
+  DEFAULT_PYTHON_VERSION: '3.12'
```

Comment:
This is not the default Python version we are going to use to build our assets, it's the version used by pdm. As such I think it should be called `PDM_PYTHON_VERSION` to avoid confusion.